src/components/User.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const User = ({ users = [] }) => {
+    return (
+        users.map(
```

Comment:
Just a syntactial change, this can be simplified to following;

```
const User = ({ users }) => (
  {users.map(user) => (
      ...
    )
  }
)
```

If you need to understand this shorter syntax please have a word with me.